### PR TITLE
Hw_4

### PR DESCRIPTION
**Pull request by andreyknyshov**
please approve my HW_4

### CODE REVIEW

File: HW_4/hw_4.2.3.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+/*3.
+Преобразовать Task 2 - 1 таким образом, чтобы значение НАПРИМЕР '2' (т.е. ЛЮБАЯ строка в которой лежат ТОЛЬКО ЦИФРЫ) пропускалось, 
+  преобразовываясь в number
+  */
+
+  const age = 10; 
+  const age_2 = 18; 
+  const age_3 = 60;
+  if (isNaN(age) != true) {
```
Comment (VladimirLQA) [inline HW_4/hw_4.2.3.js]:
```suggestion
  if (!isNaN(age)) {
```

File: HW_4/hw_4.3.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+//Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+
+
+const age = +prompt ('Enter your age: '); 
+const age_2 = 18; 
+const age_3 = 60; 
+if (isNaN(age) || (age < 1)) {
```
Comment (VladimirLQA) [inline HW_4/hw_4.3.js]:
```suggestion
if (isNaN(age)) {
```

**Review by VladimirLQA — APPROVED** [review]

File: HW_4/hw_4.2.2.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/*2.
+Преобразовать написанный код в task 1 так, чтобы сначала проверялся тип данных. 
+  И если он не number - кидалась ошибка в консоль.
+  Проверить работу кода на следующих данных 17, 18, 61, "2", "aaa"
+*/
+const age = '66'; 
+const age_2 = 18; 
+const age_3 = 60; 
+if (typeof(age) != 'number') {
```
Comment (anatoly-karpovich) [inline HW_4/hw_4.2.2.js:9]: Better make like this:
```
if (typeof(age) != 'number') {
console.log('Check the value of the ‘age’ variable. The required data type is a number');
} else {
  if (age < age_2) { 
     console.log('You don’t have access cause your age is ' + age + ' It’s less then '); 
    } else if ((age >= age_2) && (age < age_3)) { 
    console.log('Welcome  !'); 
    } else if (age > age_3) { 
     console.log('Keep calm and look Culture channel'); 
    } else console.log('Technical work'); 
}
```

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

File: HW_4/hw_4.2.2.js (hunk 3 above)
Comment (andreyknyshov) [reply HW_4/hw_4.2.2.js:9]: ok. done

**Comment by andreyknyshov** [comment]
made the corrections from the mentor

**Review by anatoly-karpovich — APPROVED** [review]

**Pull request merged**